config.go

```diff
@@ -27,10 +27,18 @@ import (
 const configVersion = "unstable"
 
 type config struct {
-	Version   string
+	Version    string
+	Generators []string
+
+	// Generator is a code generator which is used from protoc.
+	// Deprecated: Use Generators instead.
 	Generator string
-	Plugins   []string
-	Includes  struct {
+
+	// Plugins will be deprecated. It has to be per-Generator setting, but neither protoc-gen-go nor protoc-gen-go-grpc
```

Comment:
The statement about plugin support is a bit hard to understand with the double negative. Is this saying both support plugins?

Reply:
Oh. That was simply wrong. It has to be 

> but neither protoc-gen-go nor protoc-gen-go-grpc support plugins